src/main/java/appeng/me/cluster/implementations/CraftingCPUCluster.java

```diff
@@ -147,16 +146,18 @@ void addBlockEntity(CraftingBlockEntity te) {
         te.saveChanges();
         this.blockEntities.add(0, te);
 
-        if (te.isStorage()) {
-            this.availableStorage += te.getStorageBytes();
-            this.storage.add(te);
-        } else if (te.isStatus()) {
+        if (te instanceof CraftingMonitorBlockEntity) {
             this.status.add((CraftingMonitorBlockEntity) te);
-        } else if (te.isAccelerator()) {
-            if (te.getAcceleratorThreads() <= 16) {
-                this.accelerator += te.getAcceleratorThreads();
-            } else {
-                throw new IllegalArgumentException("Co-processor threads may not exceed 16 per single unit block.");
+        } else {
```

Comment:
Why else? That prevents having a storage monitor that also provides bytes and accelerator threads.

Reply:
Didn't think that combo made sense, but I guess it could be done lmao